scrape/scrape.go

```diff
@@ -90,6 +90,27 @@ var (
 			Help: "Total number of failed scrape loop reloads.",
 		},
 	)
+	targetScrapePoolTargetsLimit = prometheus.NewGaugeVec(
+		prometheus.GaugeOpts{
+			Name: "prometheus_target_scrape_pool_targets_limit",
+			Help: "Maximum number of targets allowed for this loop.",
+		},
+		[]string{"scrape_job"},
+	)
+	targetScrapePoolTargetsPostRelabeling = prometheus.NewGaugeVec(
+		prometheus.GaugeOpts{
+			Name: "prometheus_target_scrape_pool_unique_targets_post_config_relabeling",
+			Help: "Current number of unique targets remaining after config relabeling.",
+		},
+		[]string{"scrape_job"},
+	)
+	targetScrapePoolTargetsAdded = prometheus.NewGaugeVec(
+		prometheus.GaugeOpts{
+			Name: "prometheus_target_scrape_pool_targets_added",
```

Comment:
Added implies that targets are accumulating, which is not the case. This is a gauge, not a counter.